### DISCUSSION TITLE
Не подгружается медиа на твиттере

### DISCUSSION THREAD

**Discussion by MercurialPony** [hufrea/byedpi#105] [General]
Сначала скажу больше спасибо автору :D Засунул ByeDPI в докер контейнер на своем сервере и теперь подключаюсь к нему со всех устройств. Пока что все выглядит неплохо!

Но заметил, что именно на твиттере не подгружается медия типа картинок, видео, итд (pbs.twimg.com, abs-0.twimg.com, возможно еще). Использоую опции из коробки: `--split 1 --disorder 3+s --mod-http=h,d --auto=torst --tlsrec 1+s`. Что можно было бы поменять, чтобы исправить?

Через GoodbyeDPI таких проблем нет.

**Comment by MercurialPony** [comment]
Upd: поигрался сам с настройками, кажись теперь все отлично работает с этим: `--split 1 --disorder 1 --fake -1 --md5sig`. Причем работает даже быстрее, чем раньше 👌

**Comment by andruxin68** [comment]
У меня тоже не грузит медиа в твиттере. Как мне показалось, вообще не зависит от настроек. Картинки начинают загружаться внезапно. Можно несколько раз включить-выключить ВПН, но ничего не грузит. Ленту твитов просто листаешь и внезапно картинки могут начать прогружаться. Видео вообще не хочет. То есть по твиттеру при включенном ВПН доступен АПИ твиттера, потому что текст твитов загружает в любом случае, а картинки не всегда загружаются.

**Reply by MercurialPony** [reply]
Хм, ну я такого у себя не замечал. С VPN и GoodbyeDPI у меня все хорошо работает. И с вышеперечисленными настройками тоже все без проблем грузится. Но наверняка это зависит от провайдера. Также, если используешь --hosts, то надо указать все домейны твиттера (их много)
